test/test_enclave_wrangler.py

```diff
@@ -161,6 +161,46 @@ def test_upload_concept_set_version(self):
         # self.assertTrue('result' in response and not response['result'] == 'VALID')
         self.assertLess(response.status_code, 400)
 
+    # todo: after completing this 'test', create func for it in backend/db and call/assert here
+    #  - what is the ultimate goal? how many tables are we refreshing?
+    def test_get_new_objects(self):
+        """Test get_new_objects()"""
+        client = EnclaveClient()
+        # https://www.palantir.com/docs/foundry/api/ontology-resources/objects/object-basics/#filtering-objects
+        # https://unite.nih.gov/workspace/data-integration/restricted-view/preview/ri.gps.main.view.af03f7d1-958a-4303-81ac-519cfdc1dfb3
+        example_datetime = (datetime.now() - timedelta(days=1)).isoformat() + 'Z'  # works: 2023-01-01T00:00:00.000Z
+
+        # 1. Fetch data
+        # Concept set versions
+        new_csets: List[Dict] = client.get_objects_by_type(
+            'OmopConceptSet', since_datetime=example_datetime, return_type='list_dict')
+
+        # Containers
+        containers_ids = [x['properties']['conceptSetNameOMOP'] for x in new_csets]
+        container_params = ''.join([f'&properties.conceptSetId.eq={x}' for x in containers_ids])
+        new_containers: List[Dict] = client.get_objects_by_type(
+            'OMOPConceptSetContainer', return_type='list_dict', query_params=container_params, verbose=True)
+
+        # Expression items & concept set members
+        new_csets2: List[Dict] = []
+        for cset in new_csets:
+            version: int = cset['properties']['codesetId']
+            cset['expression_items'] = get_concept_set_version_expression_items(version, return_detail='full')
+            cset['members'] = get_concept_set_version_members(version, return_detail='full')
+            new_csets2.append(cset)
+
+        # 2. Database updates
```

Comment:
Based on what this gives now, I think this is the next step, but I don't know for sure what tables to do updates/deletes on: 
```
        # TODO: (i) What tables to update after this?, (ii) anything else to be done?. Iterate over:
        #  - new_containers: * new containers, * delete & updates to existing containers
        #  - new_csets2:
        #    - the cset properties itself
        #    - expression_items: we might want to delete its previous items as well, because of possible changes
        #      does expression item ID change when you change one of its flags?
        #    - members
```

Reply:
I'm not quite following this